src/components/announcer/Announcer.js

```diff
@@ -6,157 +6,110 @@
 import React from "react";
 
 import PropTypes from "prop-types";
-import classNames from "classnames";
-
-import palette from "../../util/CyVersePalette";
-
-import CheckCircleIcon from "@material-ui/icons/CheckCircle";
+import { BOTTOM, LEFT, SUCCESS, TIMEOUT, WARNING } from "./AnnouncerConstants";
 import CloseIcon from "@material-ui/icons/Close";
-import ErrorIcon from "@material-ui/icons/Error";
-import InfoIcon from "@material-ui/icons/Info";
 import IconButton from "@material-ui/core/IconButton";
-import Snackbar from "@material-ui/core/Snackbar";
-import SnackbarContent from "@material-ui/core/SnackbarContent";
-import WarningIcon from "@material-ui/icons/Warning";
-
-import { withStyles } from "@material-ui/core/styles";
-
-const variantIcon = {
-    success: CheckCircleIcon,
-    warning: WarningIcon,
-    error: ErrorIcon,
-    info: InfoIcon,
-};
+import { Snackbar, useTheme, Typography } from "@material-ui/core";
+import Alert from "@material-ui/lab/Alert";
+
+function getTextColor(theme, variant) {
+    let color;
+    switch (variant) {
+        case Error:
```

Comment:
This looks like the wrong `Error`.  I would think this should be `ERROR` from `AnnouncerConstants`

Reply:
I agree, I think this is a bug.

Reply:
An `error` occurred when importing `ERROR`